apps/wing/src/commands/test.ts

```diff
@@ -26,6 +26,7 @@ const generateTestName = (path: string) => path.split(sep).slice(-2).join("/");
  */
 export interface TestOptions extends CompileOptions {
   clean: boolean;
+  testFilter?: string;
```

Comment:
```suggestion
  filter?: string;
```

Always prefer one word if it's clear from the context.

Also, can you please add docstrings?

Reply:
Where exactly should I add docstrings?